src/cplscheme/CouplingData.cpp

```diff
@@ -132,4 +132,27 @@ void CouplingData::storeExtrapolationData()
   _extrapolation.store(values());
 }
 
+void CouplingData::clearTimeStepsStorage()
+{
+  _timeStepsStorage.clear();
+}
+
+void CouplingData::moveTimeStepsStorage()
+{
+  _timeStepsStorage.move();
+}
+
+void CouplingData::storeValuesAtTime(double relativeDt, Eigen::VectorXd data, bool mustOverrideExisting)
+{
+  PRECICE_ASSERT(math::greaterEquals(relativeDt, time::Storage::WINDOW_START), relativeDt);
+  PRECICE_ASSERT(math::greaterEquals(relativeDt, _timeStepsStorage.maxStoredNormalizedDt()), relativeDt, _timeStepsStorage.maxStoredNormalizedDt());
+  PRECICE_ASSERT(math::greaterEquals(time::Storage::WINDOW_END, relativeDt), relativeDt);
+  _timeStepsStorage.setValueAtTime(relativeDt, data, mustOverrideExisting);
+}
+
+Eigen::VectorXd CouplingData::getValuesAtTime(double relativeDt)
+{
+  return _timeStepsStorage.getValueAtTime(relativeDt);
+}
```

Comment:
`value` vs `values`?

Reply:
Will use `values`, because it is consistent with `mesh::Data::values()`.

Reply:
Done. 1acc0af70eea926c8691bd9a203e8bdb1ebc2ede